### DISCUSSION TITLE
Plugin search and Install instructions

### DISCUSSION THREAD

**Discussion by doctorfree** [codicocodes/dotfyle#66] [General]
Hi, I really like this service. Good job!

I have a couple of issues I would like to report.

First, the search for plugins used by a configuration seems to pickup references to plugins in comments.

Second, the installation instructions includes instructions on setting `NVIM_APPNAME` but my latest config got instructions to use a path:

```bash
git clone git@github.com:doctorfree/CandyVim ~/.config/doctorfree/CandyVim
NVIM_APPNAME=doctorfree/CandyVim/ nvim --headless +"Lazy! sync" +qa
```

Cloning a Neovim configuration into a subdirectory in `~/.config/<subdir>/` kinda works in 0.9 but has been disabled in Neovim nightly, if `NVIM_APPNAME` is set to a path Neovim nightly will issue an error message and exit.

So these instructions will eventually fail and even now when it appears to work, it breaks `tempname()` so any plugin using that will fail.

I've opened a discussion in the Neovim repository but so far there does not appear to be much interest in supporting this feature.

Just FYI, thanks for the excellent work.

**Comment by codicocodes** [comment]
Thank you for the comment! Sorry for the slow response I have not following up on the discussions tab!

I realized I might have had to address the install instructions since they had gotten stale on nightly. Fortunately though, it looks like Rory was able to merge a fix into nightly so it should probably be fine now - I believe!

https://github.com/neovim/neovim/pull/25233

**Reply by doctorfree** [reply]
Yes, relative paths in `NVIM_APPNAME` is now supported in nightly. Thanks for helping with that!

I think `tempname()` is still broken for subdirs in `NVIM_APPNAME` in 0.9.x until that fix is backported from nightly. But, I never noticed a problem so maybe it's rare for plugins to need that? At any rate, I'm glad we can use a subdirectory for configs now.

**Comment by codicocodes** [comment]
Regarding the plugin search picking up commented plugins,T I am doing quite simple checking for plugins based on just strings comparison and regex. I think it works well for 99% of use cases, and I am currently not convinced it would make sense to do more advanced parsing to strip out comments first. 

Would be open to a proposal/contributions on it, but still quite concerned it would increase the runtime of parsing all files in all configs (currently 800+ configs)
